e2e/src/pages/base.component.ts

```diff
@@ -0,0 +1,26 @@
+import { browser, ElementFinder, ExpectedConditions as EC } from 'protractor';
+
+export class BaseComponent {
+  url = '/';
+
+  async open(additionalPath: string = ''):Promise<void> {
```

Comment:
```suggestion
  async open(additionalPath: string = ''): Promise<void> {
```

user ctrl+alt+l / cmd+alt+l to pretify your code